app/src/main/res/values-en/strings.xml

```diff
@@ -1122,13 +1122,13 @@ If there are no signs along the whole street which apply for the highlighted sec
 
     <string name="quest_policeType_title">Which type of police station is this?</string>
 
-    <string name="quest_postboxCollectionTimes_title">What are the collection times of this postbox?</string>
-    <string name="quest_postboxCollectionTimes_resurvey_title">Are the collection times of this postbox still correct?</string>
+    <string name="quest_postboxCollectionTimes_title">What are the pickup times of this collection box?</string>
+    <string name="quest_postboxCollectionTimes_resurvey_title">Are the pickup times of this collection box still correct?</string>
```

Comment:
Sorry, having checked a bit more carefully I think this is the correct version of pick up to use:
```suggestion
    <string name="quest_postboxCollectionTimes_title">What are the pick up times of this collection box?</string>
    <string name="quest_postboxCollectionTimes_resurvey_title">Are the pick up times of this collection box still correct?</string>
```

Reply:
“Pickup” or “pick-up” is the appropriate spelling for an [attributive noun](https://en.wikipedia.org/wiki/Noun_adjunct). “Pick up” with the space is a verb.

Reply:
I went with pickup over pick-up as it's slightly more modern.  But definitely not "pick up" here.